### PR TITLE
Add AD(X) docs

### PR DESCRIPTION
**Pull request by Sporiff**
Starting off with Unity since that's the example we've received. This is the basic format for the revenue integration I think we should use. @KaihkashanAdjust could you take a look and let me know what you think?

### CODE REVIEW

**Comment by Sporiff** [comment]
> @Sporiff : Reviewed and added comments in the Vercel preview.

@KaihkashanAdjust Amazing. I'll take a look at it this week 👍

**Comment by Sporiff** [comment]
@uerceg We're still waiting for iOS/Android info on this integration, but I'd like to get your review on the initial document for Unity. We'll follow the same structure for each once we have code snippets.

https://dev-docs-git-adapp-17440-create-ad-x-dev-373c28-product-content.vercel.app/en/sdk/unity/integrations/adx

**Comment by Sporiff** [comment]
@uerceg @shashanksu this relates to [ADAPP-17408](https://adjustcom.atlassian.net/browse/ADAPP-17408) and the addition of the Ad(X) documentation. Could your team just check the information and the code snippets to make sure they look right?

- [Unity](https://dev-docs-git-adapp-17440-create-ad-x-dev-373c28-product-content.vercel.app/en/sdk/unity/integrations/adx)
- [Android](https://dev-docs-git-adapp-17440-create-ad-x-dev-373c28-product-content.vercel.app/en/sdk/android/integrations/adx)
- [iOS](https://dev-docs-git-adapp-17440-create-ad-x-dev-373c28-product-content.vercel.app/en/sdk/ios/integrations/adx)

File: src/content/docs/en/sdk/unity/integrations/adx.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+</MinorVersion>
+
+1. Create a new function that triggers when the `rewardedAd.OnPaidEvent` method is called in the AD(X) SDK. This function should receive the eCPM (Effective Cost Per Mille) amount as an argument.
+2. Instantiate an [`AdjustAdRevenue` object](/en/sdk/unity/features/ad-revenue) with the source `AdjustConfig.AdRevenueSourceAdx`.
+3. Fill in the details of your `AdjustAdRevenue` object. Divide the `ecpm` value by 1000 to return the estimated revenue for the ad.
+4. Send the ad revenue information to Adjust by calling the `trackAdRevenue` method.
+
+<Callout type="seealso">
+
+For more information about the `rewardedAd.OnPaidEvent` method, see [AD(X)'s rewarded ad documentation](hhttps://docs.adxcorp.kr/unity/sdk-integration/ad-formats/rewarded-ad/ad-x#id-4.-a-d-revenue-onpaidevent)
```
Comment (uerceg) [inline src/content/docs/en/sdk/unity/integrations/adx.mdx:20]:
```suggestion
For more information about the `rewardedAd.OnPaidEvent` method, see [AD(X)'s rewarded ad documentation](https://docs.adxcorp.kr/unity/sdk-integration/ad-formats/rewarded-ad/ad-x#id-4.-a-d-revenue-onpaidevent)
```

File: src/content/docs/en/sdk/ios/integrations/adx.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+   Adjust.trackAdRevenue(adjustAdRevenue);
+}
+```
+
+</Tab>
+<Tab title="Objective-C">
+
+```objc
+rewardedAd.paidEventHandler = ^(double ecpm) {
```
Comment (uerceg) [inline src/content/docs/en/sdk/ios/integrations/adx.mdx:39]: Where did you find the docs for this method? For [Unity](https://github.com/adjust/dev-docs/pull/410/files#diff-6f629b0da4ab06147b5859d2ffaa2c61f84e90785634d58c1f3952f83a9e23ecR27-R38), I managed to find [the docs](https://docs.adxcorp.kr/unity/sdk-integration/ad-formats/rewarded-ad/ad-x#id-4.-a-d-revenue-onpaidevent), but I haven't manage to locate the equivalent docs for iOS and Android.

Comment (Sporiff) [reply src/content/docs/en/sdk/ios/integrations/adx.mdx:39]: @uerceg The snippets were provided by Claire: https://adjustcom.atlassian.net/browse/ADAPP-17408

**Pull request merged**